### PR TITLE
Added new metadata files and added a description to the TwitchChatConnector

### PR DESCRIPTION
**Pull request by derNiklaas**
I like Pull Requests 🤷‍♂

### CODE REVIEW

File: src/main/resources/connector/org.codeoverflow.chatoverflow.requirement.service.file.fileconnector.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<connector>
+    <display>File</display>
+    <description>The file service allows you to perform file operations in the data folder. These operations include reading and writing to the data folder and creating sub-folders.</description>
+    <wiki>https://github.com/codeoverflow-org/chatoverflow/wiki/File</wiki>
```
Comment (J0B10) [inline src/main/resources/connector/org.codeoverflow.chatoverflow.requirement.service.file.fileconnector.xml:4]: That link doesn't work 😕. Could you please add the wiki page to make it working? 😉 😂

**Review by MelanX — APPROVED** [review]

File: src/main/resources/connector/org.codeoverflow.chatoverflow.requirement.service.file.fileconnector.xml (hunk 1 above)
Comment (derNiklaas) [reply src/main/resources/connector/org.codeoverflow.chatoverflow.requirement.service.file.fileconnector.xml:4]: I already created an issue regarding the missing pages (see #121 )

**Comment by MelanX** [comment]
should close #40, right?

**Comment by J0B10** [comment]
Nope, this PR only adds the metadata files to the feature branch...
To close the issue the feature branch has to be finished, tested (we all know how well chatoverflow is tested :P ) and merged...

**Pull request merged**